components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -287,6 +287,10 @@
                                     "description": "Whether only stable, latest or both versions should be warmed up. Default is stable only."
                                 }
                             }
+                        },
+                        "vmoptions": {
+                            "type": "string",
+                            "description": "Adjust VM options for the remote IDE server, especially if you want to increase the '-Xmx' memory size when working with a larger project. See https://www.jetbrains.com/help/idea/tuning-the-ide.html#configure-jvm-options"
```

Comment:
It is IntelliJ specific URL. Let's remove it from there or use product specific, i.e. for goland it shoudl be not `idea` but `go`. On website we have a template generation for it, maybe we only mention it in docs?

Reply:
I would put it simply `Configure VM options, for instance '-Xmx=4096M'` without mentioning particular IDEs or links.